### PR TITLE
[SCB-1142] Add acceptance-pack-cluster-spring-demo for alpah cluster 

### PR DESCRIPTION
**Pull request by coolbeevip**
1. Copy module acceptance-park-spring-demo and make the following modifications
2. Modify pom.xml to add two alpha server docker nodes
3. Add alpha_shutdown.btm to pack_success_scenario.feature
4. Inject alpha-shutdown.btm into the postCarBooking method of Booking trigger stop alpha master node

### CODE REVIEW

**Comment by WillemJiang** [comment]
Got the iCLA confirmation. 
https://lists.apache.org/thread.html/3e3d2e3b75f0d60541d5f1789bec9a94811305dc9856e292231c579b@%3Cprivate.servicecomb.apache.org%3E

File: alpha/alpha-core/src/main/java/org/apache/servicecomb/pack/alpha/core/actuate/endpoint/AlphaStatus.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
+package org.apache.servicecomb.pack.alpha.core.actuate.endpoint;
```
Comment (WillemJiang) [inline alpha/alpha-core/src/main/java/org/apache/servicecomb/pack/alpha/core/actuate/endpoint/AlphaStatus.java]: Please add the License header here.

File: alpha/alpha-spring-boot-compatibility/alpha-spring-boot-2-starter/pom.xml [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <parent>
+    <artifactId>alpha</artifactId>
+    <groupId>org.apache.servicecomb.pack</groupId>
+    <version>0.4.0-SNAPSHOT</version>
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+
+  <artifactId>alpha-spring-boot-starter</artifactId>
```
Comment (WillemJiang) [inline alpha/alpha-spring-boot-compatibility/alpha-spring-boot-2-starter/pom.xml]: The artifact name and version is not right.

Comment (coolbeevip) [reply alpha/alpha-spring-boot-compatibility/alpha-spring-boot-2-starter/pom.xml]: I want to express this is a different spring boot version for alpha compatibility, Do you have a naming suggestion

File: alpha/alpha-core/src/main/java/org/apache/servicecomb/pack/alpha/core/actuate/endpoint/AlphaStatus.java (hunk 1 above)
Comment (coolbeevip) [reply alpha/alpha-core/src/main/java/org/apache/servicecomb/pack/alpha/core/actuate/endpoint/AlphaStatus.java]: Added

File: alpha/alpha-spring-boot-compatibility/alpha-spring-boot-1-starter/src/main/java/org/apache/servicecomb/pack/alpha/server/actuate/endpoint/ActuatorEndpoint.java [hunk 3]
```diff
@@ -0,0 +1,38 @@
+package org.apache.servicecomb.pack.alpha.server.actuate.endpoint;
```
[the remark this answers is not in the document]
Comment (coolbeevip) [reply alpha/alpha-spring-boot-compatibility/alpha-spring-boot-1-starter/src/main/java/org/apache/servicecomb/pack/alpha/server/actuate/endpoint/ActuatorEndpoint.java]: Added

File: alpha/alpha-spring-boot-compatibility/alpha-spring-boot-1-starter/pom.xml [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+  <parent>
+    <artifactId>alpha</artifactId>
+    <groupId>org.apache.servicecomb.pack</groupId>
+    <version>0.4.0-SNAPSHOT</version>
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+
+  <artifactId>alpha-spring-boot-starter</artifactId>
+  <name>Pack::Alpha::Spring Boot 1.X Starter</name>
+  <version>1.5.17.RELEASE</version>
```
[the remark this answers is not in the document]
Comment (coolbeevip) [reply alpha/alpha-spring-boot-compatibility/alpha-spring-boot-1-starter/pom.xml]: I have changed spring version to property reference

**Pull request merged**